hebsafeharbor/identifier/entity_spliters/date_entity_splitter.py

```diff
@@ -36,9 +36,10 @@ def __call__(self, doc: Doc) -> Doc:
         birth_date_end_offsets = list(map(lambda offset: offset[0] + offset[1] - 1, birth_date_offsets))
 
         for entity in date_entities:
-            if any(entity.start - end_offset < DateEntitySplitter.WINDOW_SIZE for end_offset in birth_date_end_offsets):
+            preceding = list(filter(lambda offset: offset < entity.start, birth_date_end_offsets))
+            if any(entity.start - end_offset < DateEntitySplitter.WINDOW_SIZE for end_offset in preceding):
```

Comment:
Could use `abs(start -end)` instead?

Reply:
this can happen only if we assume that there is no chance that there will be a context word that appears after the entity and the difference will be less than the window size